src/frontend/src/styles/main.css

```diff
@@ -69,6 +69,15 @@
   font-display: swap;
 }
 
+/* Workaround font for pretending a field is a password without setting type="password"
+ * (all chars in the font are the dot symbol) */
+@font-face {
+  font-family: "password";
+  font-style: normal;
+  font-weight: 400;
+  src: url("../../assets/fonts/password.woff2") format("woff2");
```

Comment:
license wise can we use this font?

Reply:
working with a designer to get one, good catch

Reply:
made a custom font, thanks!